src/service/aws.js

```diff
@@ -0,0 +1,75 @@
+const AWS = require('aws-sdk');
+const fs = require('fs');
+const path = require('path');
+const { AWS_ACCESS_KEY_ID, AWS_BUCKET_NAME, AWS_SECRET_ACCESS_KEY } = require('../config/env');
+
+AWS.config.update({
+  accessKeyId: AWS_ACCESS_KEY_ID,
+  secretAccessKey: AWS_SECRET_ACCESS_KEY,
+});
+const s3 = new AWS.S3();
+
+var params = {
```

Comment:
This should not be declared outside the upload function scope as the values are mutated during upload.
Although unlikely this means using the same aws reference and calling upload concurrently could introduce race conditions.